README.md

```diff
@@ -17,6 +17,15 @@ To be able to start both backend and frontend locally you will need to:
 If .env file is not preset during startup of the project, it will be created using example.env file template
 The `example.env` and `.env` files are located in `apps/backend` and `apps/frontend` accordingly.
 
+## Running with docker-compose

```

Comment:
Question: I wonder if you need to adjust the comments in the https://github.com/UserOfficeProject/user-office-scheduler/blob/develop/apps/backend/example.env as well?